src/components/WeatherWidgetPlace.vue

```diff
@@ -1,12 +1,14 @@
 <template>
-  <h2 class="place">{{ props.name }}, {{ props.country }}, {{ props.date }}</h2>
+  <h2 class="place">
+    {{ props.localizedName }}, {{ props.country }}, {{ props.date }}
```

Comment:
optional. As it is purely dumb component I would pass 1 prop instead of 3 and generate it automatically as the number of props may grow.